source/mongodb/internal-database-usage.txt

```diff
@@ -70,6 +78,14 @@ App Services stores a information about unsyncable documents in the
    You can read from the ``__realm_sync.unsynced_documents`` collection,
    but you should not modify it in any way.
 
+If Device Sync is paused due to hitting the 100,000 unsyncable document limit,
+you have two options:
+   
+- :ref:`terminate and re-enable sync <terminating-realm-sync>`, even though
+   the change you're making is a non-breaking change. 
+- Contact support and request a temporary increase in the unsyncable
+   documents limit.
```

Comment:
It looks like the staged page is doing some weird formatting thing, and I wonder if it's related to the extra space here?
```suggestion
  documents limit.
```